Sources/Core/Other/TextViewMonitor.swift

```diff
@@ -0,0 +1,137 @@
+//
+//  TextViewMonitor.swift
+//  XestiMonitors
+//
+//  Created by kayeli dennis on 2018-03-27.
+//
+//  © 2018 J. G. Pusey (see LICENSE.md)
+//
+
+#if os(iOS) || os(tvOS)
+
+    import Foundation
+    import UIKit
+
+    ///
+    /// An `TextViewMonitor` instance monitors a text view for
+    /// editing-related messages.
+    ///
+
+    internal class TextViewMonitor: BaseNotificationMonitor {
+        ///
+        /// Encapsulates changes to the state of the text view.
+        ///
+        public enum Event {
+            ///
+            /// editing of the text view has begun.
+            ///
+            case didBeginEditing(UITextView)
+
+            ///
+            /// editing of the text view has ended.
+            ///
+            case didEndEditing(UITextView)
+
+            ///
+            /// the state of the text view has changed.
+            ///
+            case textDidChange(UITextView)
```

Comment:
As specified in Issue #23, this should be named `didChange(UITextView)`. Please reorder the cases as necessary to maintain alphabetical order.